### PR TITLE
feat(pattern): cta added to content-block-segmented items

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#1845 

### Description

`CTA` can now be included in the `ContentBlockSegmented` items,
A unit test was created and the documentation was updated to reflect this change.

A spacing update was done in order to reflect the latest design specs for `xlg` breakpoint.

### Changelog

**New**

- CTA can now be used on items

**Changed**

- SCSS Spacing updates

### CODE REVIEW

**Comment by jeffchew** [comment]
@RaphaelAmadeu can you fix the merge conflicts?

**Comment by asudoh** [comment]
@RaphaelAmadeu My apologies for my massive changes to stories. Given most of the changes were of indents I think the merge effort is straightforward, but please ping me if you have any questions, etc. Thanks!

File: packages/react/src/patterns/blocks/ContentBlockSegmented/README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Add the following line in your `.env` file at the root of your project.
 | --------- | -------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `heading` | YES      | String    | Short copy describing content item.                                                                                                                             |
 | `image`   | NO       | Object    | See the [`Image`](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/components/Image) component for full usage details. |
+| `cta`     | NO       | Object    | See the [`CTA`](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/components/CTA) component for full usage details.     |
```
Comment (jeffchew) [inline packages/react/src/patterns/blocks/ContentBlockSegmented/README.md:98]: Wonder if we should say somewhere here that only `jump` and `local` are allowed types?

Comment (RaphaelAmadeu) [reply packages/react/src/patterns/blocks/ContentBlockSegmented/README.md]: @jeffchew I've added this to the `README` file

**Pull request merged**